class ResourceNotFoundException {
  final String message;

  const ResourceNotFoundException([this.message = 'O CEP n√£o existe!']);
}
